samples/go/nomdex/Readme.md

```diff
@@ -0,0 +1,55 @@
+# Nomdex 
+
+Nomdex demonstrates how Noms maps can be used to index values in a database and provides a simple query language to search for objects.
+
+## Description
+This program experiments with using ordinary Noms maps as indexes. It leverages the fact that Maps in Noms are implemented by prolly-trees which are a type of B-Tree. They are balanced and require relatively few accesses to find any leaf node which makes them ideal for use as indexes.
```

Comment:
I don't think "type of B-Tree" is strictly correct. More correct "which have similar characteristics to B-Tree: they are balanced, require ..."